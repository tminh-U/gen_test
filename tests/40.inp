14
10 9 5 3 2 11 13 11 8 16 18 17 10 11 
10
6 10
6 7
4 9
2 12
5 7
9 11
11 13
2 14
11 14
10 10
