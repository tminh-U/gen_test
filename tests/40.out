14
4
21
204
8
8
4
246
6
2
